buildpipeline/Core-Setup-Publish.json

```diff
@@ -314,10 +429,23 @@
     },
     "TeamName": {
       "value": "DotNetCore"
+    },
+    "VsoToolsRepo": {
```

Comment:
Anything that is defined as a variable in a VSTS build definition, should be prefixed with `PB_`.  We haven't fully made this true everywhere.  The idea is to prevent them from leaking into builds as environment variables which can make repro'ing failures difficult.